src/Datadog.Trace/Datadog.Trace.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We keep switching back and forth between having and not having the UTF-8 BOM. Why does the tool even modify this file?

Reply:
The Versions replacement logic does this. You suggested we add this option to the versioning tool when it was implemented: `new UTF8Encoding(encoderShouldEmitUTF8Identifier: false)`

Reply:
Oh, yeah. I remember. I wonder what keeps adding it, then. Visual Studio? VSCode? Maybe it would be easier to always add the BOM instead of removing it? Level of care is low, as long as we can be consistent and avoid the extra whitespace noise in commit diffs.